README.md

```diff
@@ -158,9 +158,9 @@ You can find the documentation of this module in [ClientUI](client-ui/README.md)
 Add a dependency on Spark Agent.
 ```xml
 <dependency>
-    <groupId>za.co.absa.spline</groupId>
-    <artifactId>spark-agent</artifactId>
-    <version>0.4.0</version>
+    <groupId>za.co.absa.spline.agent.spark</groupId>
```

Comment:
Do you plan to have more than Spark agents? If so, maybe the groupId should end at `za.co.absa.spline.agent`

Reply:
I started with that, but quickly realized that the agent project itself could be (and it actually is) a multi-module project. Providing that there might be more types of agents, one in it's own project with several modules it could quickly lead to a mess in the artifact IDs. Prefixing each and every module with the agent type seems cumbersome.